charts/shield/values.yaml

```
@@ -206,7 +206,23 @@ host:
   # The node selector for the host shield
   node_selector: {}
   # The tolerations for the host shield
-  tolerations: []
+  tolerations:
```

```suggestion
  tolerations: # +doc-gen:break
```
And re-run `make docs`